content/efcore/getting-started/samples/configuring-deletion/manual-deletion-strategy/README.md

```diff
@@ -0,0 +1,12 @@
+# Manual Deletion Strategy
+
+This code example demonstrates how deletion is configured in EF Core using the OnDelete method and its parameters: Cascade Delete, SetNull, and Restrict.
```

Comment:
Actually, let's call this example "Deletion Configuration", and update the description to just say "This code example demonstrates how to manually configure the deletion behavior for EF Core using Fluent API."